src/Validation/Validation.php

```diff
@@ -1424,6 +1424,17 @@ public static function isArray($value)
     }
 
     /**
+     * Check that the input value is a scalar.
+     *
+     * @param mixed $value The value to check
+     * @return bool
+     */
+    public static function isScalar($value)
+    {
+        return is_scalar($value);
```

Comment:
Imho it should NOT behave different than `is_scalar()`; But we can have `isScalarOrNull()` or `isScalarOrNullish()` (latter accepts '' and null)?